src/settings.py

```diff
@@ -1,11 +1,30 @@
 import json
+import logging
+from enum import Enum
 from pathlib import Path
 from typing import Any, Dict, List, Tuple
 
 from pydantic import BaseModel, BaseSettings, HttpUrl
 from pydantic.env_settings import SettingsSourceCallable
 
 
+class LogFormat(Enum):
+    GENERIC = "generic"
+    JSON = "json"
+
+
+class LogLevel(Enum):
```

Comment:
I'm surprised this enum does already exist in pythons built-in `logging` module

Reply:
Does it?